modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,132 @@
+For example, the URI for a GET method for an endpoint named `/customers` in a particular specification might have looked like this with version 1 of the mocking service:
++
+`+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575/api/customers+`
++
+Version 2 of the mocking service changes the URI to this and uses a different ID:
++
+`+https://anypoint.mulesoft.com/mocking/api/v1/links/62a7d6af-ebc7-43fa-a815-e415e329f2ff/api/customers+`
+
+. If any third-party tools are testing endpoints in the API specification for that project, replace the previous base URI with the new base URI in the links that those tools are using.
++
+For example, suppose that this URI is being used in a tool:
++
+`+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575/api/customers+`
++
+You would need to replace `+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575+` with `+https://anypoint.mulesoft.com/mocking/api/v1/links/{new-ID}+`, where `new-ID` is the ID that version 2 of the mocking service generates.
++
+You can get the new base URI from version 2 of the mocking service after step 1 above.
+
+== New Features in Version 2 That You Can Use in API Designer
+
+Version 2 of the mocking service, which was released on January 10, 2019, introduced the following features that you can use in API Designer.
+
+=== Support for OAS 2.0
+
+You can now test endpoints that are in API specifications that are written in OAS 2.0.
+
+
+=== Behavioral Headers
+
+Add headers to your simulated calls to methods, changing the behavior of the mocking service in response to those calls The following new headers are available. See xref::apid-behavioral-headers.adoc[Add Behavioral Headers to Simulated API Calls] for full details.
+
+* MS2-Delay: Indicates how many milliseconds the request should take to answer.
+* MS2-Error-Level: Indicates which range of error status codes the mocking service is restricted to use.
+* MS2-Error-Rate: Indicates the probability of getting an error status code as a response.
+* MS2-Randomize: When this header is set to true, the mocking service auto-generates a payload response if no example is defined for the response.
+* MS2-Status-Code: Specifies the status code or codes that the mocking service can choose from for the return message.
+* MS2-Strict-Model-Validations: When you set this header to false, you can use the mocking service even when the API specification contains errors
+
+=== Performance Improvements
+
+The mocking service uses fewer resources, and responds to requests in less time on average.
+
+
+=== Authenticated Access to API Specifications That Are Shared on Exchange
+
+Only Anypoint Platform users can simulate calls to your API specification after it is published to a private instance of Exchange, whether they are simulating calls in Exchange or using third-party tools. The mocking service now uses a base URI in Exchange that differs from the one that you can test with in API Designer.
```

Comment:
ditto GLOBAL comment about "third-party"

Reply:
Is the first sentence true for Version 1 but no longer for Version 2, or is it still true but modified by Version2?
It's unclear to me how the second sentence relates to the first; does the new base URI change the behavior indicated in the first?